tasks/create.yaml

```diff
@@ -45,8 +45,18 @@ tasks:
   - name: pepr-build
     description: "Build the UDS Core Pepr Module"
     actions:
+      - description: "Set custom pepr image if registry1 flavor"
+        cmd: |
+          if [ "${FLAVOR}" = "registry1" ]; then
+            echo "--custom-image registry1.dso.mil/ironbank/opensource/defenseunicorns/pepr/controller:v0.27.0"
```

Comment:
Couldnt this be done in one line in the pepr build step instead of a separate action with a multiline if statement and setVariable?